ambari-server/src/examples/rack_hosts.yaml

```diff
@@ -0,0 +1,108 @@
+racks:
+  racka-1:
+    hostGroups:
+    - subnet: 120.190.211.64/15
```

Comment:
I can't find where subnets are used? If they're not used was it intentional or missed by accident?

Reply:
They are not used and came by accident. Will remove it.